### DISCUSSION TITLE
Comfy.org join them

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/Fooocus#3187] [Ideas]
Fooocus is ComfyUı backned, why don't we join their discord server. Share our ideal, ask questins, learn New techniques. Union will leap forwarded Fooocus.

**Comment by IPv6** [comment]
Fooocus not relying on Comfy, actually

**Reply by wisdombox44** [reply]
Thanks

**Reply by mashb1t** [reply]
@IPv6 *not fully, but using parts of their code.
There's also a huge discussion for Forge, if you want to read into this topic, but be prepared for beef between comfyanonymous and lllyasviel: https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/169
